test/e2e/kubectl.go

```diff
@@ -0,0 +1,215 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+)
+
+const (
+	nautilusImage       = "kubernetes/update-demo:nautilus"
+	kittenImage         = "kubernetes/update-demo:kitten"
+	updateDemoSelector  = "name=update-demo"
+	updateDemoContainer = "update-demo"
+)
+
+var _ = Describe("kubectl", func() {
+
+	runUpdateExample()
+})
+
+func runUpdateExample() {
+	updateDemoRoot := filepath.Join(root, "examples/update-demo")
+
+	It(fmt.Sprint("should create and stop a replication controller"), func() {
```

Comment:
Why use a fmt.Sprint here? It takes a string as the first argument and this just adds an extra level of unnecessary indirection. 

It("should create and stop a replication controller", func() { ...})

is much nicer.